Tasks/DockerV1/containerbuild.ts

```diff
@@ -66,5 +66,11 @@ export function run(connection: ContainerConnection): any {
         context = tl.getPathInput("buildContext");
     }
     command.arg(context);
+
+    command.on("stdout", data => {
+        let taskOutputPath = utils.writeTaskOutput("build", data);
+        tl.setVariable("DockerOutputPath", taskOutputPath);
```

Comment:
I would like to know if there is a way for testing this. I tried using `process.env` but the variable was not set there even though it was set correctly in the code ( if you print `tf.stdoutput` you can see that the variable was set correctly). I also didn't see an example of a test for the regular output variable `DockerOutput`

Reply:
I don't know a place where output variable is being tested.
The [documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#setvariable-initialize-or-modify-the-value-of-a-variable) says the variable is available as env variable to the tasks that follow.
Maybe test that the command to set the variable is present in `stdout`?

Also, can you pls test it out by actually running in a pipeline and adding a script to output this to the logs?